### DISCUSSION TITLE
Border-radius style override

### DISCUSSION THREAD

**Discussion by Syncretik** [TanStack/table#4432] [Q&A]
Is there a way to adjust the `border-radius` style on the table? doesn't seem to be working when I try styling the `table` element I'm using.

**Comment by mohamedtsx** [comment]
We cannot adjust border radius to HTML table tags directly. Instead, apply the radius to `<td>` cells by targeting the first and last child cells if you're styling a row. If you're targeting the table itself, you can use additional containers.

```jsx
<TableHead
  key={header.id}
  className="text-start first:rounded-s-xl last:rounded-e-xl"
>
  {header.isPlaceholder ? null : flexRender(
    header.column.columnDef.header,
    header.getContext()
  )}
</TableHead>

**Comment by farrukhkomilovich** [comment]
Border radiusini to'g'ridan-to'g'ri @tanstack/react-table ga moslashtira olmaymiz. Buning o'rniga, agar biz qatorni(columns) shakllantirayotgan bo'lsak, birinchi va oxirgi bola katakchalarni belgiilab olish orqali <td> katakchalarida radiusni qo'llashimiz mumkin. Border-radiusni Tablening o'ziga qilmoqchi bo'lsak, qo'shimcha konteynerlar(div)dan foydalanishimiz mumkin.
